static/js/i18n/strings/en/stats_var_labels.json

```diff
@@ -1062,5 +1062,9 @@
   "AirQualityIndex_AirPollutant": {
     "defaultMessage": "AQI",
     "description": "Label used to describe statistical variables in chart legends, string should be short. Describes the air quality index."
+  },
+  "DifferenceRelativeToBaseDate2006_Daily_Max_Temperature_RCP85": {
+    "defaultMessage": "Max Temperature Change",
```

Comment:
please keep "RCP 8.5"  and base year 2006 in the string

Reply:
Done